zookeeper/uninstalled.sls

```diff
@@ -28,13 +28,3 @@ zookeeper-reload-systemctl:
     - onchanges:
       - file: {{ zk.systemd_unit }}
 {%- endif %}
```

Comment:
This thing just checks when the file changes. For example, when the file content changes or the file is deleted, then the state will be executed. But the state will be executed only once.